src/components/select/MultiSelectConnected.tsx

```diff
@@ -136,7 +138,7 @@ export class MultiSelectConnected extends React.Component<IMultiSelectProps, {}>
 
     private getButton(props: ISelectButtonProps): JSX.Element {
         const classes = classNames('multiselect-input', {'mod-sortable': this.props.sortable});
-        const buttonAttrs = this.props.selected && this.props.selected.length === this.props.items.length ? {disabled: true} : {};
+        const buttonAttrs = {disabled: !this.props.noDisabled && this.props.selected && this.props.selected.length === this.props.items.length};

```

Comment:
Use the new logic with the old way to add the attribute to avoid `disabled: false` on the element